lib/vagrant-libvirt/action/handle_box_image.rb

```diff
@@ -80,14 +80,17 @@ def call(env)
             message << " in storage pool #{config.storage_pool_name}."
             @logger.info(message)
 
+            @storage_volume_uid = storage_uid env
+            @storage_volume_gid = storage_gid env
```

Comment:
can you make these local variables rather than instance ones.

Reply:
The patch that introduced the feature and caused the problem removed them as follows:

-                @storage_volume_uid = storage_uid env
-                @storage_volume_gid = storage_gid env

so i thought it would be good to re-introduce them like before. Anyway, thats up to you :)

Reply:
fair enough, I'd forgotten that they were instance variables before